connect/cli/core/utils.py

```diff
@@ -1,9 +1,13 @@
 # -*- coding: utf-8 -*-
```

Comment:
These functions are specific of your plugins right ? If so it does not make sense to put them in core/utils.py

Reply:
Right now they are only used by the new translation plugin, but as I saw that this same code is repeated in other plugins, I moved them to core to allow those plugins to use them. I could do this in a later PR.